openml-lightgbm/lightgbmlib_build/pom.xml

```diff
@@ -0,0 +1,21 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.feedzai.openml.lightgbmlib</groupId>
+  <artifactId>lightgbmlib</artifactId>
+  <version>2.3.1</version>
+  <packaging>jar</packaging>
+  <url>git@gitlab.feedzai.com:data-science/make-lightgbm.git</url>
```

Comment:
Se my more general comment on the PR global discussion tab, but it doesn't make sense for public repo to reference FZ internal repos.

Reply:
Me and Sheng already discussed this: he's going to open source that repository as well, and then update all the files in this folder accordingly

Reply:
I think we can close this for now.
When the `make-lightgbm` becomes opensourced, we can update this folder.
WDYT?